packages/bundler/src/es6-rewriter.ts

```diff
@@ -68,11 +68,18 @@ export class Es6Rewriter {
     // resolutions for such things as bare module specifiers.
     const jsImportResolvedUrls = new Map<string, Map<string, ResolvedUrl>>();
     if (document) {
-      jsImportResolvedUrls.set(input, this._getJsImportResolutions(document));
+      jsImportResolvedUrls.set(input, this.getEs6ImportResolutions(document));
     }
     const rollupBundle = await rollup({
       input,
-      external,
+      external: (moduleId, parentId) => {
```

Comment:
Add comment like "external tells rollup not to bundle a particular import".

Reply:
Comment'd!